app/components/carbon-calculator.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="footprint rounded-3">
-  <p class="note">Note: The limit of carbon foot print usable is 60 tons</p>  
+  <p class="note">Note: The limit of carbon foot print usable is 60 tons</p>
```

Comment:
make the note as Threshold for carbon footprint emission is 60 kg CO2e per month

Reply:
made the change